tests/phpunit/CRM/Activity/BAO/ActivityTest.php

```diff
@@ -592,7 +586,6 @@ public function testGetActivitiesforContactSummary() {
       'activity_type_id' => NULL,
       'offset' => 0,
       'rowCount' => 0,
-      'sort' => NULL,
```

Comment:
I removed this by accident but I think it might not be a bad thing as it ensures enotices don't get thrown